app/controllers/api/v1/merchants/search_controller.rb

```diff
@@ -0,0 +1,17 @@
+class Api::V1::Merchants::SearchController < ApplicationController
+  def index
+    @merchant = Merchant.where(nil)
```

Comment:
this controller is bloated, will be a candidate for refactoring in further iterations